ring/ring.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
Do we need to add a mention about Dapr editing the file? I don't know what the convention is here.

Reply:
That's done 3 lines down. I assumed the editor being Dapr was assumed through context.

Reply:
Yeah, I saw, was just wondering if the name Dapr needs to be mentioned, I genuinely don't know if that's the case.